Assets/Mirage/Runtime/SocketLayer/Peer.cs

```diff
@@ -266,7 +266,14 @@ private void ReceiveLoop()
         private void HandleMessage(Connection connection, Packet packet)
         {
             // ingore message of invalid size
-            if (!packet.IsValidSize()) { return; }
+            if (!packet.IsValidSize())
+            {
+                if (logger.filterLogType == LogType.Log)
+                {
+                    logger.Log($"Receive from {connection} was not too small");
```

Comment:
`logger.Log($"Receive from {connection} was not of correct size");`

Reply:
`IsValidSize` only checks minimum size, but I've fixed the typo, should read "was too small"